invenio/modules/oauthclient/contrib/orcid.py

```diff
@@ -115,24 +115,26 @@
 
 def account_info(remote, resp):
     """ Retrieve remote account information used to find local user. """
-
     from ..handlers import token_session_key
 
-    # Store ORCID of the user in session for later retrieval
-    session[token_session_key(remote.name) + "_orcid"] = resp.get("orcid")
-    return dict(email=None, nickname=None)
+    account_info = dict(email=None, nickname=None, orcid=resp.get("orcid"),
+                        external_id=resp.get("orcid"), external_method="orcid")
+
+    # Store account info of the user in session for later retrieval
+    session[token_session_key(remote.name) + "_accountinfo"] = account_info
```

Comment:
@jmartinm I would go for the same suffix like the variable name, hence WDYT about replacing `_accountinfo` with `_account_info`.